src/NuGet.Core/NuGet.DependencyResolver.Core/PackagingUtility.cs

```diff
@@ -10,19 +10,20 @@ public static class PackagingUtility
         public static LibraryDependency GetLibraryDependencyFromNuspec(Packaging.Core.PackageDependency dependency)
         {
             // Start with the default
-            var includeType = LibraryIncludeType.Default;
+            var includeType = ~LibraryIncludeFlags.ContentFiles;
```

Comment:
Perhaps, it will be better if we put the default value as a static or const on LibraryIncludeFlagUtils and use that